### DISCUSSION TITLE
Notifications not being read unless you press "Back to notifications"

### DISCUSSION THREAD

**Discussion by CaedenPH** [community/community#34852] [Other Feature Feedback, Questions, & Ideas] [answered]
On mobile, the notifications work perfectly. You click on a notification, and if you go back to the notifications bar this notification is read.

However, on desktop, you can't just go back. Github make's you press the "Back to notifications" button in order to mark it as read. I strongly dislike this, as it wastes time going through notifications.

**Comment by CaedenPH** [comment, marked as answer]
Turns out a reload is required to unread a notification, which pressing "back to notifications" does.

**Reply by dodas** [reply]
> Turns out a reload is required to unread a notification, which pressing "back to notifications" does.

This seems illogical.
